voucher/src/main/java/org/devcourse/voucher/console/Console.java

```diff
@@ -0,0 +1,21 @@
+package org.devcourse.voucher.console;
+
+import java.util.Scanner;
+
+public class Console implements Input, Output{
+    public static final String CREATE = "create";
+    public static final String LIST = "list";
+    public static final String EXIT = "exit";
```

Comment:
아마 명령에 대한 문자열 정보인 것 같은 데 Console 객체가 알고 관리하는 게 맞는 지 의문이 들어요!

Reply:
```suggestion
    public static final String CREATE_COMMAND_STRING = "create";
    public static final String LIST_COMMAND_STRING = "list";
    public static final String EXIT_COMMADN_STRING = "exit";
```
정답은 없지만 제가 제시한 예시처럼 조금 더 의미 있는 변수명을 사용하면 좋을 것 같아요!